README.md

```diff
@@ -6,5 +6,21 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+
+Rails is an application development framework. it exists under an assumption that there is a best way to develop a web app. It creates a framework with what it assumes every developer needs to build an app. It is used to help build web apps!!
+
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
Your diagram is mostly correct. Just one quibble. The controller doesn't pass the request on to the model - rather, it attempts to handle the request itself by invoking the appropriate model methods. The model never sees the request itself.